### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by yepzdk**
#### Link to ticket

https://jira.itkdev.dk/browse/KLIMA-61

#### Description

Possibility of configuring a footer with an image and background color.

#### Screenshot of the result

Screenshot 2023-05-04 at 07 51 48

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: assets/react/components/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const Wrapper = styled.div`
+  background-color: var(--background-color, hsl(0deg 0% 100%));
+  filter: var(--filter-shadow);
+  width: 100vw;
+  height: calc((100vh / var(--total-rows)) * var(--footer-height));
+  display: grid;
+  place-content: center;
+
+  @media (orientation: portrait) {
+    /* flex-direction: column-reverse;
```
Comment (tuj) [inline assets/react/components/footer.jsx]: Maybe remove? Or leave comment about why commented out code is here.

File: assets/react/controllers/Mosaic.jsx [hunk 2]
```diff
@@ -137,10 +134,17 @@ function Mosaic() {
         }
 
         const randomTiles = variant.randomTiles ?? true;
+        const gridColumns = screen.gridColumns ?? 6;
+        const totalRows = screen.gridRows ?? 5;
+        const gridRows = variant.footerHeight ? totalRows - variant.footerHeight : totalRows ?? 5;
+        const numberOfTiles = gridColumns * gridRows;
+
+        console.log(variant.footerHeight);
```
Comment (tuj) [inline assets/react/controllers/Mosaic.jsx]: Remove

**Review by tuj — CHANGES_REQUESTED** [review]

**Review by tuj — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -37,3 +37,41 @@ docker run --rm -v .:/app --workdir /app node:18 yarn install
...
+| gridExpose            | 1-9           | `"gridExpose":2` |
+| exposeShowBorder      | true,false    | `"exposeShowBorder":false` |
+| exposeShowIcon        | true,false    | `"exposeShowIcon":false` |
+| exposeTimeout         | 1-9           | `"exposeTimeout":7` |
+| exposeFontSize        | xs,s,m,l,xl   | `"exposeFontSize":"m"` |
+| Logo |
+| mosaicLogo            | true,false    | `"exposeShowIcon":false` |
+| Footer |
+| footerHeight          | 1-9           | `"footerHeight":"1"` |
+| footerImageSrc        | Url           | `"https://pathtoimagesource.io/footerimage.svg"` |
```
Comment (tuj) [inline README.md:65]: "footerImageSrc":"https://pathtoimagesource.io/footerimage.svg"

File: README.md [hunk 4]
```diff
@@ -37,3 +37,41 @@ docker run --rm -v .:/app --workdir /app node:18 yarn install
...
+| footerBackgroundColor | Hex format    | `"footerBackgroundColor":"#F4DCEA"` |
+| Cta box |
+| ctaBoxTitle           | Text          | `"ctaBoxTitle":"Del dit billede"` |
+| ctaBoxDescription     | Text          | `"ctaBoxDescription":"Skan koden og indsend dit bidrag"` |
+| ctaBoxImage           | Url           | `"https://pathtoimagesource.io/ctaimage.svg"` |
```
Comment (tuj) [inline README.md:70]: "ctaBoxImage":"https://pathtoimagesource.io/ctaimage.svg"

**Review by tuj — COMMENTED** [review]
Found two missing keys in the examples

File: README.md (hunk 4 above)
Comment (yepzdk) [reply README.md:70]: Not sure what you mean? The example includes both. Although it is different paths.

**Pull request merged**